### ISSUE TITLE
Broken Travis at the moment

### ISSUE THREAD

**Issue by terenc3t** [prysmaticlabs/prysm#2187]
Travis is currently broken with the following errors

```
# github.com/prysmaticlabs/prysm/contracts/deposit-contract/sendDepositTx
/home/travis/.gimme/versions/go1.11.7.linux.amd64/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: /tmp/go-link-485517972/000014.o: unrecognized relocation (0x2a) in section `.text'
/usr/bin/ld: final link failed: Bad value
collect2: error: ld returned 1 exit status
```

**Comment by prestonvanloon** [comment]
Root cause: https://github.com/golang/go/issues/31293

**Comment by nisdas** [comment]
Resolved by #2215
